docs/source/app-dev/app-arch.rst

```diff
@@ -81,13 +81,13 @@ your project, e.g. ``yarn add @daml/react``. Please explore the ``create-daml-ap
 to see the usage of these libraries.
 
 To make your life easy when interacting with the ledger, the DAML assistant can generate
-corresponding typescript data definitions for the data types declared in the deployed DAR.
+corresponding TypeScript data definitions for the data types declared in the deployed DAR.
```

Comment:
The codegen does not only generate types but also JavaScript objects containing metadata about the types and templates in the DALF. We should mention that here.

Reply:
I'll update the description of daml2js in the next PR. This one is only about the typings.

Reply:
I'm just working on the next PR, but I wouldn't mention the metadata here. This chapter was ment to be as simple an accesible as possible.

Reply:
I'm not sure there's a way around mentioning the metadata. The JavaScript part of the generated code doesn't contain anything bit metadata. You don't need to go into any sort of details, just say something like "the generated JavaScript contains metadata about the types and templates in the DAML package" and that's it.